### PR TITLE
sending some metrics to prometheus pushgateway

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

**Comment by ghost** [comment]
Build failed.

- [thoth-coala ](https://zuul.thoth-station.ninja/logs/02/102/152a093bf438a7af209e4bbf933be41dc5435361/check/thoth-coala/b7cd1ef/report.html) : RETRY_LIMIT in 1m 40s

**Comment by goern** [comment]
recheck

**Comment by goern** [comment]
recheck

**Comment by goern** [comment]
recheck

**Comment by goern** [comment]
recheck

**Comment by goern** [comment]
recheck

**Review by fridex — APPROVED** [review]
:+1:

File: app.py [hunk 1]
```diff
@@ -48,12 +62,16 @@ def graph_refresh(graph_hosts: str = None, graph_port: int = None) -> None:
...
+            _METRIC_PACKAGES_ADDED.inc()
+
             packages.append(f"{package}=={version}\n")
 
         for dependent_package, dependent_versions in graph.retrieve_dependent_packages(package).items():
             for dependent_version in versions:
-                _LOGGER.info(f"Adding dependency refresh  {dependent_version!r}=={dependent_version!r} "
+                _LOGGER.info(f"Adding dependency refresh {dependent_version!r}=={dependent_version!r} "
                              f"from {package}=={version}")
+                _METRIC_PACKAGES_ADDED.inc()
```
Comment (fridex) [inline app.py]: Maybe one thing - we could distinguish between the dependent ones and the ones that were actually added. This way we will see if there is "explosion" in scheduling we would like to minimize.

File: app.py [hunk 2]
```diff
@@ -65,6 +83,7 @@ def graph_refresh(graph_hosts: str = None, graph_port: int = None) -> None:
                 solver=solver, debug=_LOG_SOLVER, packages=package, output=_SOLVER_OUTPUT
             )
             _LOGGER.info("Scheduled solver %r for package %r, pod id is %r", solver, package, pod_id)
+            _METRIC_SOLVERS_SCHEDULED.labels(solver).inc()
```
Comment (fridex) [inline app.py:91]: Maybe one thing - we could distinguish between the dependent ones and the ones that were actually added. This way we will see if there is "explosion" in scheduling we would like to minimize.

File: app.py (hunk 1 above)
Comment (fridex) [inline app.py]: Maybe one thing - we could distinguish between the dependent ones and the ones that were actually added. This way we will see if there is "explosion" in scheduling we would like to minimize.

**Comment by goern** [comment]
recheck

**Pull request merged**